src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
we can just return the result of the `if` condition here :)
```suggestion
        return user != null && user.getPassword().equals(password);
```

Reply:
Understood, I even forgot about this option, need more practice, thank you